tests/acceptance/file-details/dynamic-scan-test.js

```
@@ -819,243 +819,327 @@ module('Acceptance | file-details/dynamic-scan', function (hooks) {
+      this.server.get('/v2/files/:id/dynamicscans', (schema, req) => {
+        const { limit, mode, engine, group_status } = req.queryParams || {};
+
+        assert.strictEqual(engine, '2');
+        assert.strictEqual(group_status, 'running');
+
+        const results = schema.dynamicscans
+          .where({
+            file: req.params.id,
+            ...(mode ? { mode: Number(mode) } : {}),
+          })
+          .models.slice(0, limit ? Number(limit) : results.length);
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)